Watchman.DomainModel/Users/WarnEvent.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Watchman.DomainModel.Messages;
+using Watchman.Integrations.MongoDB;
+
+namespace Watchman.DomainModel.Users
+{
+    public class WarnEvent : Entity
+    {
+        public User Receiver { get; private set; }
```

Comment:
czemu to ma w sobie obiekt User, User to agregat

Reply:
już usunąłem usera w ogóle